bin/base-ns-id-gen.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 SUB_MAP=( "aat=DCD-CNP-DEV"
-  "perftest=DCD-CNP-QA"
+  "demo=DCD-CNP-DEV"
```

Comment:
Are we meant to be committing these changes?

Reply:
local change 😄